patches/minecraft/net/minecraft/client/renderer/ItemInHandRenderer.java.patch

```diff
@@ -24,6 +24,14 @@
              boolean flag1 = CrossbowItem.m_40932_(p_109377_);
              boolean flag2 = humanoidarm == HumanoidArm.RIGHT;
              int i = flag2 ? 1 : -1;
+@@ -384,6 +_,7 @@
+             this.m_109322_(p_109372_, p_109377_, flag2 ? ItemTransforms.TransformType.FIRST_PERSON_RIGHT_HAND : ItemTransforms.TransformType.FIRST_PERSON_LEFT_HAND, !flag2, p_109379_, p_109380_, p_109381_);
+          } else {
+             boolean flag3 = humanoidarm == HumanoidArm.RIGHT;
++            if (net.minecraftforge.client.extensions.common.IClientItemExtensions.of(p_109377_).applyForgeHandTransform(p_109379_, f_109299_.f_91074_, humanoidarm, p_109377_, p_109373_, p_109378_, p_109376_)) {} else
```

Comment:
This can be simplified into a `if (!condition)` without the `else`. Additionally, it may be good to put a `// FORGE: ` marker.

Reply:
It is to skip any other vanilla transformations and go straight into rendering, and that's why there's `else` there
It is meant to be `else if`, combined with if statement right after it.

Reply:
The same can be achieved with `if (!condition)`, because it will take the whole `if-else` chain below as the one statement guarded by the `if (!condition)`.

As in, instead of:
```java
if (condition) {} else
if (first) {
	// ...
} else if (second) {
	// ...
}
```

We can instead do:
```java
if (!condition)
if (first) {
	// ...
} else if (second) {
	// ...
}
```

The above can be reformatted to look like the following (with the braces in comments further illustrating the point:
```java
if (!condition) /* { */
	if (first) {
		// ...
	} else if (second) {
		// ...
	}
/* } */
```